event-publisher/src/test/resources/uk/gov/ons/ctp/common/event/PackageFixture.FulfilmentEvent.json

```diff
@@ -1,7 +1,7 @@
 [
   {
     "header": {
-      "version": "v0.3_RELEASE",
```

Comment:
You missed some 0.4 changes needed in other elements below.

Also look at `FirestoreEventPersistenceTest.FulfilmentEvent.json` where another is also missed